src/pages/python/using-python-for-web-development/index.md

```diff
@@ -3,4 +3,4 @@ title: Using Python for Web Development
 ---
 Python is known for its beautiful syntax and ease of readability. This scripting language can be used for some quick-and-dirty prototyping of your idea of a web-app. It can also be used to build scalable and maintainable large web applications.
 
-So let's get started with some web frameworks and learn how they help build web app on-hands. But before that, we would need to set-up the development environment by completing <a href='http://forum.freecodecamp.com/t/setting-up-python-web-framework-django-and-flask/19354' target='_blank' rel='nofollow'>installation of web frameworks in Python</a>.
\ No newline at end of file
+So let's get started with some web frameworks and learn how they help build web app on-hands. But before that, we would need to set-up the development environment by completing <a href='https://guide.freecodecamp.org/python/setting-up-python-web-framework-django-and-flask' target='_blank' rel='nofollow'>installation of web frameworks in Python</a>.
```

Comment:
Hi @joostsijm 

Thanks for fixing the url where the link goes to. Can you please change it to Markdown instead of the current HTML formatting? All links should be in Markdown.